Configurations/descrip.mms.tmpl

```diff
@@ -104,7 +104,7 @@ BLDDIR={- $config{builddir} -}
 # to testing.
 VERBOSE=$(V)
 
-VERSION={- "$config{major}.$config{minor}.$config{patch}$config{prerelease}$config{build_metadata}" -}
+VERSION={- "$config{full_version}" -}
```

Comment:
**Side note:** Looking at this line an idea comes to my mind: wouldn't it make more sense and be more consistent with older conventions, if you would change the names as follows (in a separate pr)?

```
version -> version_number
full_version  -> version
```

and

```
OPENSSL_VERSION_STR -> OPENSSL_VERSION_NUMBER_STR 
OPENSSL_FULL_VERSION_STR -> OPENSSL_VERSION_STR
```
Now it would still be ok change it IMO.

Reply:
I think that it would only be confusing to reinstate `version_number` /  `OPENSSL_VERSION_NUMBER` (even with `_STR` appended), since that name is closely connected to the calculated version integer.  I really really really don't want to go back there.